crates/aptos/src/update/update_helper.rs

```diff
@@ -48,7 +48,10 @@ pub fn build_updater(
     let target = format!("{}-{}", arch_str, target);
 
     let install_dir = match install_dir.clone() {
-        Some(dir) => dir,
+        Some(dir) => {
```

Comment:
I suggested formatting improvements but these sort of look like debug prints, is it intentional that this and the one for install dir are left in the code?

Reply:
yeah, it is for debugging, removed the print statement